### ISSUE TITLE
People merge dialog stays blank sometimes

### ISSUE THREAD

**Issue by gloriousDan** [nextcloud/photos#1423]
**Describe the bug**
This bug doesn't happen every time but sometimes. I haven't been able to connect what could cause it.

It occurs when trying to merge different people in the people tab. Usually when selecting _Merge with different person_, a dialogue appears where you can select the person to merge it with. Now sometimes the dialogue doesn't show up and the rest of the screen is just darkened until I click somewhere else or press escape. This can be fixed by a reload of the site. When selecting _Merge with different person_ after the reload it takes a couple seconds for it to load the different people in the dialogue.

**To Reproduce**
1. Go to Photos
2. Click on People
3. Select a Person which you want to merge
4. Click on the 3 dots on the top right corner
5. Select _Merge with different person_
6. Now sometimes (as often as every second time) the dialogue doesn't show up.

**Expected behavior**
The dialogue should show and allow me to select a person to merge the person with

**Desktop (please complete the following information):**
 - OS: Arch Linux, on Kernel 6.0.2-arch1-1 running Plasma 5.26.1 with wayland
 - Browse: Mozilla Firefox 106.0.1
 - Version 
   - nextcloud: 25.0.0
   - Photos: 2.0.0

**Browser log**
```
Content Security Policy: Couldn't process unknown directive 'frame-ancestors'self'' 2
InstallTrigger is deprecated and will be removed in the future. constants.js:50:14
GEThttps://<my-domain>/apps/photos/css/icons.css?v=d233662f-19
[HTTP/2 404 Not Found 90ms]

The resource from “https://<my-domain>/apps/photos/css/icons.css?v=d233662f-19” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff). faces
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
JQMIGRATE: Migrate is installed, version 3.4.0 jquery-migrate.min.js:2:698
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:60:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:60:15
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
OCA.Files.Sidebar initialized Sidebar.js:36:10
OCA.Files.Sidebar initialized sidebar.js:39:8
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
Proxying an event bus of version 2.1.1 with 1.3.0 major.js:2:14
Proxying an event bus of version 2.1.1 with 3.0.0 index.js:19:12
OCA.Sharing.ShareSearch initialized ShareSearch.js:33:10
OCA.Sharing.ExternalLinkActions initialized ExternalLinkActions.js:33:10
OCA.Sharing.ExternalShareActions initialized ExternalShareActions.js:33:10
Proxying an event bus of version 2.1.1 with 3.0.2 index.esm.js:11:20
Proxying an event bus of version 2.1.1 with 3.0.2 index.js:19:20
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
Proxying an event bus of version 2.1.1 with 3.0.0 index.js:19:12
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:60:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:60:15
OCA.Comments.View initialized comments-app.js:32:8
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:60:15
Proxying an event bus of version 2.1.1 with 3.0.2 ProxyBus.ts:16:20
Proxying an event bus of version 2.1.1 with 3.0.2 ProxyBus.ts:16:20
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:60:15
Proxying an event bus of version 2.1.1 with 3.0.2 jsonp chunk loading:1:58
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
[WARN] viewer: Some mimes were ignored because they are not enabled in the server previews config 
Object { level: 2, app: "viewer", uid: "<user>", ignoredMimes: (3) […] }
ConsoleLogger.js:60:18
OCA.Viewer initialized Viewer.js:51:25
Proxying an event bus of version 2.1.1 with 3.0.2 index.js:19:20
oc_appswebroots is deprecated: use OC.appswebroots instead, this will be removed in Nextcloud 20 2 globals.js:60:15
Proxying an event bus of version 2.1.1 with 3.0.2 index.esm.js:11:12
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:6
Proxying an event bus of version 2.1.1 with 3.0.2 index.js:19:12
Proxying an event bus of version 2.1.1 with 3.0.2 ProxyBus.ts:16:12
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:6
Proxying an event bus of version 2.1.1 with 3.0.2 ProxyBus.ts:16:12
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:60:15
Proxying an event bus of version 2.1.1 with 3.0.2 ProxyBus.ts:16:20
Proxying an event bus of version 2.1.1 with 3.0.2 ProxyBus.ts:16:20
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:60:15
Registering notifications container as a menu NotificationsApp.vue:217
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:60:15
Notifications permissions granted NotificationsApp.vue:442
Polling interval updated to 30000 NotificationsApp.vue:402
Started background fetcher as session_keepalive is enabled NotificationsApp.vue:264
Proxying an event bus of version 2.1.1 with 1.3.0 index.es.js:2337:14
Proxying an event bus of version 2.1.1 with 3.0.0 index.js:19:12
session heartbeat polling started session-heartbeat.js:103:9
Got notification data NotificationsApp.vue:365
Polling interval updated to 30000 NotificationsApp.vue:402
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:60:15
Proxying an event bus of version 2.1.1 with 3.0.2 NcRelatedResourcesPanel.js:2:91876
SW registered:  
ServiceWorkerRegistration { installing: null, waiting: null, active: ServiceWorker, navigationPreload: NavigationPreloadManager, scope: "https://<my-domain>/apps/photos", updateViaCache: "imports", onupdatefound: null, pushManager: PushManager }
Photos.vue:166
InstallTrigger is deprecated and will be removed in the future. 2 constants.js:50:14
InstallTrigger is deprecated and will be removed in the future. 2 constants.js:50:14
No new notification data received NotificationsApp.vue:370
Polling interval updated to 30000 NotificationsApp.vue:402
InstallTrigger is deprecated and will be removed in the future. 2 constants.js:50:14
TypeError: t.faceDetections.find(...) is undefined
    getFaceCover FaceCoverMixin.js:53
    getFaceCover FaceCoverMixin.js:52
    cover FaceCover.vue:93
    VueJS 3
    coverDimensions FaceCover.vue:97
    get VueJS
vue.runtime.esm.js:2989:37
InstallTrigger is deprecated and will be removed in the future. 2 constants.js:50:14
TypeError: t.faceDetections.find(...) is undefined
    getFaceCover FaceCoverMixin.js:53
    getFaceCover FaceCoverMixin.js:52
    getCoverStyle FaceCoverMixin.js:76
    H FaceMergeForm.vue:1
    Xe VueJS
    H FaceMergeForm.vue:1
    VueJS 3
vue.runtime.esm.js:2989:37
No new notification data received NotificationsApp.vue:370
Polling interval updated to 30000 NotificationsApp.vue:402
```
The `t.faceDetections.find(...)` error occurs everytime the bug occurs

**Comment by marcelklehr** [comment]
Thank you @gloriousDan for reporting this!

I've been able to reproduce this and wrote up a fix over at #1457

**Comment by gloriousDan** [comment]
Thank you very much @marcelklehr for the fast fix and PR.
This will make it a lot easier to use it :).
Overall I really love the possibility to recognize faces within nextcloud so thank you a lot for your work on this
